zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -65,7 +69,7 @@ object Server {
       zExec          <- UnsafeChannelExecutor.make[R].toManaged_
       channelFactory <- ZManaged.access[ServerChannelFactory](_.get)
       eventLoopGroup <- ZManaged.access[EventLoopGroup](_.get)
-      httpH           = ServerRequestHandler(zExec, settings.http)
+      httpH           = ServerRequestHandler(zExec, settings.http, settings.errorHandler)
```

Comment:
```suggestion
      httpH           = ServerRequestHandler(zExec, settings)
```

Reply:
settings is a private case class for server object

Reply:
we can make it `private [zhttp]`